WebDriverAgentLib/FBAlert.m

```diff
@@ -123,7 +123,14 @@ - (BOOL)acceptWithError:(NSError **)error
       withDescriptionFormat:@"Failed to find accept button for alert: %@", alertElement]
      buildError:error];
   }
-  return [defaultButton fb_tapWithError:error];
+  [defaultButton tap];
```

Comment:
1) You trigger tap twice, which might bring unexpected behaviour. 
2) There is no tests that covers that issue.